pkg/apis/pipeline/v1alpha1/pipeline_validation.go

```diff
@@ -58,5 +59,17 @@ func (ps *PipelineSpec) Validate() *apis.FieldError {
 			}
 		}
 	}
+
+	if ps.Timeout != nil {
+		// timeout should be a valid duration of between 0 and 24 hours.
+		maxTimeout := time.Duration(24 * time.Hour)
```

Comment:
i think declaring this at the top of the file as a `var` is consistent with how we declare other values like this

Reply:
could even be a constant :angel:

Reply:
I've yanked this out anyway. =)